src/hotspot/share/opto/divnode.cpp

```diff
@@ -1379,37 +1397,49 @@ const Type* ModLNode::Value(PhaseGVN* phase) const {
 
 
 //=============================================================================
-//------------------------------Value------------------------------------------
-const Type* ModFNode::Value(PhaseGVN* phase) const {
-  // Either input is TOP ==> the result is TOP
-  const Type *t1 = phase->type( in(1) );
-  const Type *t2 = phase->type( in(2) );
-  if( t1 == Type::TOP ) return Type::TOP;
-  if( t2 == Type::TOP ) return Type::TOP;
+//------------------------------Idealize---------------------------------------
+Node *UModLNode::Ideal(PhaseGVN *phase, bool can_reshape) {
```

Comment:
```suggestion
Node* UModLNode::Ideal(PhaseGVN* phase, bool can_reshape) {
```
Ah, maybe you did not mean to touch it, but on GitHub it looks like you did... Maybe you just reordered things.

Reply:
Makes it a little trickier to review though.

Reply:
Yes, I think I just re-ordered this without realizing.